boards/arm/nucleo_wb55rg/doc/nucleowb55rg.rst

```diff
@@ -242,18 +246,11 @@ You should see the following message on the console:
 Debugging
 =========
 
-While STM32WB55RG is not yet supported you can debug an application using pyocd.
-Here is an example for the :ref:`hello_world` application.
-Start pyocd gdbserver on your machine:
-
-.. code-block:: console
-
-   $ pyocd gdbserver
-
-Then launch debug on your board:
+You can debug an application in the usual way.  Here is an example for the
+:ref:`blinky-sample` application.
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
+   :zephyr-app: samples/basic-blinky
```

Comment:
basic-blinky? Maybe samples/basic/blinky

Reply:
yep, good catch.  The sample is in ``samples/basic/blinky``

Reply:
Txs, fixed.